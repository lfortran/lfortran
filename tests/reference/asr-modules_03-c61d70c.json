{
    "basename": "asr-modules_03-c61d70c",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/modules_03.f90",
    "infile_hash": "e8a8f67348a67ae303e75cebd603e3d97bb6eb8737fd17d7ade3b3e0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-modules_03-c61d70c.stdout",
    "stdout_hash": "76fcec35d6d5b1104666308bbfa5111fae15209b6e21e638296891c4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}