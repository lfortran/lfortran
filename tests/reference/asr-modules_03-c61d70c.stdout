(TranslationUnit 
    (SymbolTable
        1
        {
            access_vars:
                (Module 
                    (SymbolTable
                        2
                        {
                            print_vars:
                                (Function 
                                    (SymbolTable
                                        3
                                        {
                                            
                                        }) 
                                    print_vars 
                                    [] 
                                    [(Print 
                                        () 
                                        [(StringConstant 
                                            "priv = " 
                                            (Character 1 7 () [])
                                        )
                                        (Var 2 priv)] 
                                        () 
                                        ()
                                    )
                                    (Print 
                                        () 
                                        [(StringConstant 
                                            "publ = " 
                                            (Character 1 7 () [])
                                        )
                                        (Var 2 publ)] 
                                        () 
                                        ()
                                    )] 
                                    () 
                                    Source 
                                    Public 
                                    Implementation 
                                    () 
                                    .false. 
                                    .false. 
                                    .false. 
                                    .false. 
                                    .false. 
                                    [] 
                                    [] 
                                    .false.
                                ), 
                            priv:
                                (Variable 
                                    2 
                                    priv 
                                    Local 
                                    (RealConstant 
                                        1.500000 
                                        (Real 4 [])
                                    ) 
                                    () 
                                    Save 
                                    (Real 4 []) 
                                    Source 
                                    Private 
                                    Required 
                                    .false.
                                ), 
                            publ:
                                (Variable 
                                    2 
                                    publ 
                                    Local 
                                    (RealConstant 
                                        2.500000 
                                        (Real 4 [])
                                    ) 
                                    () 
                                    Save 
                                    (Real 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    access_vars 
                    [] 
                    .false. 
                    .false.
                )
            
        }) 
    []
)
